src/shared/reducers/challenge-listing/index.js

```diff
@@ -812,7 +812,7 @@ function create(initialState) {
     sorts: {
       ongoing: 'updated',
       openForRegistration: 'updated',
```

Comment:
Apply same logical to openForRegistration

Reply:
Done